### PR TITLE
Add `magic` field in getversion

### PR DESCRIPTION
**Pull request by Tommo-L**
Fix https://github.com/neo-project/neo-modules/issues/358

### CODE REVIEW

File: src/RpcClient/Models/RpcVersion.cs
Comment (roman-khimov) [inline src/RpcClient/Models/RpcVersion.cs:32]: useragent?

Comment (Tommo-L) [reply src/RpcClient/Models/RpcVersion.cs]: Fixed https://github.com/neo-project/neo-modules/pull/363/commits/99819ccda95807a6a91d7877e939aafa484aa42e

**Review by shargon — APPROVED** [review]

**Comment by devhawk** [comment]
Can this please wait until after #335 is merged?

**Comment by superboyiii** [comment]
Test Pass. It works well on mine.
```
            Console.WriteLine("**********RpcClient*************");
            RpcClient rpcClient = new RpcClient("http://localhost:12332");
            Console.WriteLine();

            /*getversion*/
            Console.WriteLine("********************************");
            Console.WriteLine("getversion:");
            RpcVersion getversion = rpcClient.GetVersion();
            var result = getversion.ToJson();
            Console.WriteLine("getversion: " + result);
            Console.WriteLine("********************************");
            Console.ReadLine();
```
image
image

**Review by superboyiii — APPROVED** [review]

**Pull request merged**
